feature-libs/quote/core/facade/quote.service.spec.ts

```diff
@@ -266,6 +267,33 @@ describe('QuoteService', () => {
       });
   });
 
+  it('should call addQuoteComment command when called with empty string a entry number', () => {
+    service
+      .addQuoteComment(mockQuote.code, mockComment, '')
```

Comment:
since the entrynumber is optional, is the '' parameter even needed?

Reply:
the previous test  is testing the case without the optional parameter. Here we want to explicitly test the behavior for an empty string.